cmake/GoogleCloudCppCommon.cmake

```diff
@@ -86,6 +86,7 @@ elseif (GOOGLE_CLOUD_CPP_GENERATE_DOXYGEN)
         # Any additional includes that the library needs
         ${GOOGLE_CLOUD_CPP_DOXYGEN_EXTRA_INCLUDES})
     set(DOXYGEN_GENERATE_LATEX NO)
+    set(DOXYGEN_HAVE_DOT NO)
```

Comment:
Is there a benefit to actually having dot?

Reply:
I think it makes the diagrams that we see at the top of this page, for example: https://googleapis.dev/cpp/google-cloud-common/HEAD/group__guac.html

I think we tried to turn it off with our other settings too but doxygen has since evolved.